public/js/draw/draw.js

```diff
@@ -432,7 +434,7 @@ function finishRegion() {
 
     if (mode === 'region-mode' && curPath !== null && curPath.elbows.length >= 3) {
         curPath.setAttributeNS(null, 'd', curPath.getAttribute('d') + 'Z');
-        curPath.setAttributeNS(null, 'data-group', nodeColourId);
```

Comment:
I didn't notice this earlier, but I believe this old attribute was used in the CSS to actually fill in the regions with different colours. Look inside the css files to see if there are any rules that can be removed.